### PR TITLE
Updated all in-repo references and scripts to Python 3

### PR DESCRIPTION
**Pull request by brenthuisman**
bors try

### CODE REVIEW

**Comment by haampie** [comment]
bors try-

**Comment by haampie** [comment]
bors try

File: scripts/tsplot [hunk 1]
```diff
@@ -84,7 +82,7 @@ def parse_clargs():
     return P.parse_args(argsbis)
 
 def isstring(s):
-    return isinstance(s,str) or isinstance(s,unicode)
+    return isinstance(s,str) or isinstance(s,str)
```
Comment (bcumming) [inline scripts/tsplot:85]: This should simplify to just
```Python
    return isinstance(s,str)
```
To the point where you can just call `isinstance(s,str)` directly inline.

Comment (brenthuisman) [reply scripts/tsplot]: I used 2to3, I removed this and some other clutter in introduced.

**Pull request merged**